src/eko/gamma.py

```diff
@@ -10,7 +10,7 @@
 
 
 @nb.njit(cache=True)
-def gamma_0():
+def gamma_as1():
```

Comment:
I was thinking that maybe it's more clear if we drop this convention and we just rename beta_as_2aem1 -> beta_qcd_as2aem1. @AleCandido @giacomomagni what do you think?

Reply:
fine with me - as said before this is actually the fully explicit name

Reply:
Essentially fine, but for `beta` actually `beta_as` it is even more explicit than `beta_qcd` (because it is the derivative of `as` in the first place), while `gamma_as` makes not so much sense, `gamma_qcd` is much more meaningful (since it's coming from the contribution of QCD interaction).

If you want to stick to only one of the two (because at the end the theory and the coupling are one-to-one), then prefer `qcd`, since `gamma_as` is way less meaningful.

Reply:
I would go for `gamma_qcd` as well.